src/sdk/namespace_cache.js

```diff
@@ -257,7 +263,11 @@ class NamespaceCache {
                             xattr: object_info_hub.xattr,
                             last_modified_time: (new Date(object_info_hub.create_time)).getTime(),
                         };
-                        return this.namespace_nb.upload_object(upload_params, object_sdk);
+                        const upload_res = await this.namespace_nb.upload_object(upload_params, object_sdk);
+                        this._update_metric('set_cache_write_bytes', object_info_hub.first_range_data.length);
+                        this._update_metric('set_cache_object_read_miss_count', 1);
+                        this._update_metric('set_cache_object_read_count', 1);
+                        return upload_res;
```

Comment:
Why do we need the update_metric indirection? why not just:
```
this. stats_collector. set_cache_write_bytes(`object_info_hub.first_range_data.length`);
this.stats_collector.set_cache_object_read_miss_count(1);
this.stats_collector.set_cache_object_read_count(1);
```
It is much more verbose, less prone to the string changing as the js linter should know what is the interface of the report class (see my comments on the `endpoint_report` changes)

Please note that this comment should also apply to all the other this._update_metric class in this file

Reply:
@paulxuzhang are you ok with this request?

Reply:
Pushed changes.